client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -1,5 +1,197 @@
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import ReduxFormRadio from 'components/redux-forms/redux-form-radio';
+import ReduxFormSelect from 'components/redux-forms/redux-form-select';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import ReduxFormToggle from 'components/redux-forms/redux-form-toggle';
+import SectionHeader from 'components/section-header';
+
+const JobSubmission = ( { duration, translate } ) => {
+	return (
+		<div>
+			<form>
+				<SectionHeader label={ translate( 'Account' ) }>
+					<FormButton compact primary />
+				</SectionHeader>
+				<Card>
+					<FormFieldset>
+						<ReduxFormToggle
+							name="job_manager_user_requires_account"
+							text={ translate( 'Require an account to submit listings' ) } />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Limits job listing submissions to registered, logged-in users.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_enable_registration"
+							text={ translate( 'Enable account creation during submission' ) } />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Includes account creation on the listing submission form, to allow ' +
+								'non-registered users to create an account and submit a job listing simultaneously.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_generate_username_from_email"
+							text="Generate usernames from email addresses" />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Automatically generates usernames for new accounts from the registrant\'s ' +
+								'email address. If this is not enabled, a "username" field will display instead.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+
+					<FormFieldset>
+						<FormLabel>
+							{ translate( 'Role' ) }
+						</FormLabel>
+						<ReduxFormSelect name="job_manager_registration_role">
+							<option value="editor">{ translate( 'Editor' ) }</option>
+							<option value="author">{ translate( 'Author' ) }</option>
+							<option value="contributor">{ translate( 'Contributor' ) }</option>
+							<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+							<option value="teacher">{ translate( 'Teacher' ) }</option>
+							<option value="employer">{ translate( 'Employer' ) }</option>
+							<option value="customer">{ translate( 'Customer' ) }</option>
+							<option value="shop_manager">{ translate( 'Shop manager' ) }</option>
+						</ReduxFormSelect>
+						<FormSettingExplanation>
+							{ translate( 'Any new accounts created during submission will have this role.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+				</Card>
+			</form>
+
+			<form>
+				<SectionHeader label={ translate( 'Approval' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Waiting approval' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpjm-job-submission)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).